### DISCUSSION TITLE
Default scroll width smooth-scrollbar.js

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [idiotWu/smooth-scrollbar#491] [Q&A]
How to combine a strip with a normal element scroll?
For example, if I set it to overflow: scroll, after scrolling page I try to scroll element in bottom - nothing happens no matter how hard I try.

**Comment by dolphin-wood** [comment]
Have you set `continuousScrolling` to `true`?

**Reply by MyNameTony1000** [reply]
The point here is that the plugin is initialized on the page, but not on the block I'm talking about. There, the scroll is made through a fixed size and overflow: scroll. So the question arose - is it possible to combine such approaches or is the only option to initialize the plugin for all blocks with a scroll?

**Reply by dolphin-wood** [reply]
A native scrollbar inside the smooth-scrollbar? We got a tricky solution here: https://github.com/idiotWu/smooth-scrollbar/issues/92#issuecomment-324981807

**Reply by MyNameTony1000** [reply]
thank you very much)) Didn't see this discussion)

**Reply by MyNameTony1000** [reply]
> A native scrollbar inside the smooth-scrollbar? We got a tricky solution here: [#92 (comment)](https://github.com/idiotWu/smooth-scrollbar/issues/92#issuecomment-324981807)

Wasn't there a better solution that doesn't stop the container with the triggered strip from scrolling when the mouse is on the block with overflow : scroll? It's just that if there are several such blocks with a native strip and they occupy the entire width, it turns out a very large space on which the scroll of the main block does not work
